gcs/depends

```diff
@@ -0,0 +1,3 @@
+lightdm-gtk-greeter
+gecosws-artwork
```

Comment:
Why does this package depend on gecosws-artwork?

Reply:
Because gtk greteer depends on Gecos theme and it belongs to gecosws-artwork package.

Reply:
Ok